src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -128,12 +130,18 @@
   //  this is the high-level control of the tabulation algorithm
   // returns: set containing winner(s)
   Set<String> tabulate() throws TabulationCancelledException {
-    logSummaryInfo();
-
     if (config.needsRandomSeed()) {
-      TieBreak.setRandomSeed(config.getRandomSeed());
+      if (config.getTiebreakMode() == TieBreakMode.GENERATE_PERMUTATION) {
+        // sort candidate permutation for reproducibility
+        Collections.sort(config.getCandidatePermutation());
+        Collections.shuffle(config.getCandidatePermutation(), new Random(config.getRandomSeed()));
```

Comment:
It feels a little weird that we instantiate a second `Random` here. Maybe `Tabulator` should just have this as a member variable and pass it to the constructor of `TieBreak`?

Shouldn't matter in practice, but it feels like it centralizes the random logic a bit more clearly.

Reply:
Done.